wrappers/ortools.rb

```diff
@@ -645,21 +660,20 @@ def parse_output(vrp, _services, points, _matrix_indices, _cost, _iterations, ou
           {
             original_service_id: service.original_id,
             service_id: service_id,
-            type: service.type.to_s,
             point_id: service.activity ? service.activity.point_id : service.activities.collect{ |activity| activity[:point_id] },
             detail: service.activity ? build_detail(service, service.activity, service.activity.point, nil, nil, nil) : { activities: service.activities }
           }
         } + (vrp.shipments.collect(&:id) - collected_indices.collect{ |index| index >= vrp.services.size && ((index - vrp.services.size) / 2).to_i < vrp.shipments.size && vrp.shipments[((index - vrp.services.size) / 2).to_i].id }.uniq).collect{ |shipment_id|
           shipment = vrp.shipments.find{ |sh| sh.id == shipment_id }
           [{
             original_shipment_id: shipment.original_id,
-            shipment_id: shipment_id.to_s,
+            pickup_shipment_id: shipment_id.to_s,
             type: 'pickup',
             point_id: shipment.pickup.point_id,
             detail: build_detail(shipment, shipment.pickup, shipment.pickup.point, nil, nil, nil)
           }, {
             original_shipment_id: shipment.original_id,
-            shipment_id: shipment_id.to_s,
+            delivery_shipment_id: shipment_id.to_s,
```

Comment:
I asked a question earlier about this but it is buried under the review I am afraid.